### PR TITLE
Update Command_op.java

### PR DESCRIPTION
**Pull request by DUFCLiam**
#953

### CODE REVIEW

**Comment by OnlineRepz** [comment]
Remove -c and -s parts and it will be fine

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 1]
```diff
@@ -26,6 +26,16 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             playerMsg("Correct usage: /opall");
             return true;
         }
+         if (args[0].equalsIgnoreCase("-c") || args[0].equalsIgnoreCase("-s"))
+        {
+            if (TFM_AdminList.isSuperAdmin(sender) || senderIsConsole) {
+            playerMsg("Correct usage: /opall [-c|-s]");
+            return true;
+        } else {
```
Comment (OnlineRepz) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Formatting

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 2]
```diff
@@ -26,6 +26,16 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             playerMsg("Correct usage: /opall");
             return true;
         }
+         if (args[0].equalsIgnoreCase("-c") || args[0].equalsIgnoreCase("-s"))
+        {
+            if (TFM_AdminList.isSuperAdmin(sender) || senderIsConsole) {
```
Comment (OnlineRepz) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Formatting

**Comment by JeromSar** [comment]
@DUFCLiam PR is on hold, associated issue is not accepted.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 3]
```diff
@@ -51,8 +61,15 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+         if (!player.isOp())
+         {
+         	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
+         	player.setOp(true);
+         }
+         else
+         {
+         	playerMsg("You don't have permission to OP users that are already Opped.", ChatColor.YELLOW);
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Message here doesnt make sense or is even needed. Just display no message at all...

**Comment by Wild1145** [comment]
@JeromSar Re-Open?

**Pull request closed without merging**